Makefile

```diff
@@ -1,5 +1,19 @@
 TERRAFORM_VERSION:=$(shell cat example/.terraform-version)
 
+define generate-example
+	docker run \
+		--interactive \
+		--tty \
+		--volume $(shell pwd):/src \
+		--workdir /src/example \
+		--entrypoint /bin/sh \
+		hashicorp/terraform:$(1) \
+			-c \
+				"terraform init && \
+				terraform plan -out tfplan && \
+				terraform show -json tfplan > tfplan.json"
+endef
```

Comment:
This is a NOOP change moving this macro higher in the Makefile, for formatting purposes.